src/main/java/training/day3/Day3.java

```diff
@@ -4,100 +4,135 @@
 import java.util.Scanner;
 
 public class Day3 {
+    //Homework
+    //Zadacha 1
+    public static void zadacha1(int first, int second, int third) {
+        int min = first;
+        if (second < min) {
+            min = second;
+        }
+        if (third < min) {
```

Comment:
That is OK. Can also try it with else-if, meaning:
```java
        if (second < min) {
            min = second;
        }
        else if (third < min) {
            min = third;
        }
``` 
If the first statement is false, it will got check in the else-if statement.